src/browser/screens/Equipment.react.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import Component from 'react-pure-render/component';
+import classNames from 'classnames';
+
+import ScreenImage from '../components/ScreenImage.react';
+import Item from '../components/CompassItem.react';
+import JobDescription from '../components/JobDescription.react';
+import boundScroll from '../lib/boundScroll';
+import ChevronIcon from '../components/ChevronIcon.react';
+
+import data from '../data/equipment.json'
+
+if (process.env.IS_BROWSER) {
+  require('./Equipment.styl');
+}
+
+@boundScroll()
+export default class Compass extends Component {
+  static propTypes = {
+    actions: React.PropTypes.object.isRequired,
+    candidate: React.PropTypes.object.isRequired
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {currentTab: {id: 'tools'}};
+  }
+
+  proceed() {
+    const {actions: {nextScreen}} = this.props;
+    nextScreen();
+  }
+
+  renderList() {
+    let id = this.state.currentTab.id;
+    return (
+      data[id].map(element =>
+        <li key={element}>
+          {element}
+        </li>
+      )
+    )
+  }
+
+  selectTab(id) {
+    this.setState({
+      currentTab: {...this.state.currentTab, id}
+    })
+  }
+
+  render() {
+    const equipmentUrl = {
+      img: "../../../assets/img/equipment.svg"
+    };
```

Comment:
To mi nie daje spokoju. W kodzie widze, ze raz podajesz w dol nazwe klasy, raz img. Z drugiej strony widze, ze w css kazdy screen ma swoja klase, wiec moze wszystko ujednolicic i podawac w dol nazwe klasy i tam deklrowac bg. Co o tym myslisz?

Reply:
ScreenImage jest ostylowany jednakowo dla każdego ekranu, przy definicji źródła zawsze przekazywany obraz jest jako props data.img, a przekazywany obraz to nie background tylko img.